Hearthstone Deck Tracker/Enums/ShowDateOnDecksOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace Hearthstone_Deck_Tracker.Enums
+{
+	public enum ShowDateOnDecksOptions
```

Comment:
This is a bit of a convoluted name. Something like `DateType` makes it more clear what this is and makes it more "reusable".